### PR TITLE
[#445] Added PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by s0b0lev**

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=446) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=446) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=446) it.</sub>

**Comment by Xndr7** [comment]
Hi. 
Maybe you could refer to [this](https://github.com/sendgrid/open-source-library-data-collector/blob/master/.github/PULL_REQUEST_TEMPLATE.md) file for more details that can be added to the template. 
Since the maintainers suggested it in #445. 
Cheers! 👍

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+**Description of the change**:
+
+**Reason for the change**:
+
```
Comment (mbernier) [inline .github/PULL_REQUEST_TEMPLATE]: Can you please add a couple more things? @xndr7 noted, this is a really good example -
 you can copy/pasta it here if you would like! 
https://github.com/sendgrid/open-source-library-data-collector/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Comment (mbernier) [reply .github/PULL_REQUEST_TEMPLATE]: #456 resolves this comment.

**Comment by thinkingserious** [comment]
Hello @s0b0lev,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**